pkg/cmd/certregenerationcontroller/cmd.go

```diff
@@ -123,10 +124,22 @@ func (o *Options) Run(ctx context.Context) error {
 		return err
 	}
 
+	kubeInformers := informers.NewSharedInformerFactory(kubeClient, 10*time.Minute)
```

Comment:
Not sure if this is a good idea to piggyback on certregen controller, but I'd prefer to avoid running another binary and modifying static pod with a new container